src/ci/bin/test.inc.sh

```diff
@@ -405,7 +210,6 @@ cromwell::private::create_build_variables() {
     CROMWELL_BUILD_DOCKER_TAG="${CROMWELL_BUILD_DOCKER_TAG:0:128}"
     CROMWELL_BUILD_DOCKER_TAG="${CROMWELL_BUILD_DOCKER_TAG//[^a-zA-Z0-9.-]/_}"
 
-    CROMWELL_BUILD_REQUIRES_SECURE="${CROMWELL_BUILD_REQUIRES_SECURE-false}"
```

Comment:
The "secure" concept did not make the leap during the Travis to Github transition